pytorch_lightning/trainer/trainer.py

```diff
@@ -76,6 +76,63 @@ class Trainer(
     TrainerDeprecatedAPITillVer0_9,
     TrainerDeprecatedAPITillVer0_10,
 ):
+    """
+    Example:
+
+        >>> import torch
+        >>> from torch.nn import functional as F
+        >>> from torch.utils.data import Dataset, DataLoader
```

Comment:
if you want, you can move these to the top of the file, they don't necessarily need to be inside the doctest.

Reply:
but they are not called outside test flake8 may complain about unused impprts